osal/src/os/posix/src/os-impl-idmap.c

```diff
@@ -20,61 +20,17 @@
 
 /**
  * \file     os-impl-idmap.c
- * \ingroup  posix
- * \author   joseph.p.hickey@nasa.gov
+ * \ingroup  FreeRTOS
  *
  */
 
 /****************************************************************************************
                                     INCLUDE FILES
  ***************************************************************************************/
 
-#include "os-posix.h"
-#include "bsp-impl.h"
```

Comment:
Should this be removed?

Reply:
posix should be left in, bsp-impl removed as per previous comment.